paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -147,11 +148,23 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
             }
         }
 
+        val supportedPaymentMethodTypes = async {
```

Comment:
@tillh-stripe I added this here since I needed to update some tests. See [here](https://github.com/stripe/stripe-android/pull/5975/files#diff-ea471356277e87a41365593550e06ddfdc6262e5a9a6a83d5d0a21bafe1df1b3R671) for the new tests that were added.